### PR TITLE
can configure 'includeRoutees' for akka agent

### PR DESCRIPTION
**Pull request by hughsimpson**

### CODE REVIEW

**Comment by janm399** [comment]
Good; how can you test that the change actually works as expected? You've introduced a new feature, but there's no test or GH issue for it.

**Comment by hughsimpson** [comment]
Running the current tests should've thrown a ConfigException if it wasn't working, shouldn't it? Our declaration of 
protected boolean includeRoutees = ...
should've broken if we weren't correctly getting the Config...

**Comment by janm399** [comment]
Hugh, can you take a look at this PR and get it to a stage where it can be merged?

File: test/src/main/scala/org/eigengo/monitor/testcounter.scala [hunk 1]
```diff
@@ -126,7 +128,7 @@ object TestCounterInterface {
       (x.aspect == aspect) &&
       (tagFilter match {
         case AnyTag           => true
-        case ExactTag(tag)    => x.tags.size == 1 && x.tags.head == tag
+        case SingleTag(tag)    => x.tags == tag :: Nil
```
Comment (janm399) [inline test/src/main/scala/org/eigengo/monitor/testcounter.scala]: One last tiny thing: can you align the `=>` with the line above? Then it's looking good.

**Pull request merged**